modules/View.js

```diff
@@ -56,6 +56,72 @@
       if (!options.noActivate) {
         this.activate();
       }
+      // this.updateDelegateEvents();
+    },
+
+
+    /**
+    * called by updateDelegateEvents to wrap callback functions of event calls
+    * with event triggers
+    * @param method {function} callback method of event
+    * @param eventName {String} description of event type 
+    * @return modified mehtod with before/after event triggers
+    * @method trackEvents
+    */
+    trackEvents : function(method, eventName){
```

Comment:
Consider renaming this to be more general and also indicate that this is not part of the public API with double "__". Something like: '__wrapDelegatedEvents' or '__triggerDomEventHelpers' or something that's less "tracking" - that's just one application of these triggered events